apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -19,7 +19,11 @@ const { Meter } = require('@condo/domains/meter/utils/serverSchema')
 const { connectContactToMeterReading } = require('@condo/domains/meter/utils/serverSchema/resolveHelpers')
 const { addClientInfoToResidentMeterReading } = require('@condo/domains/meter/utils/serverSchema/resolveHelpers')
 const { addOrganizationFieldPlugin } = require('@condo/domains/organization/schema/plugins/addOrganizationFieldPlugin')
-const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RESIDENT, SERVICE } = require('@condo/domains/user/constants/common')
+
+function canEditBillingStatusFields ({ authentication: { item: user } }) {
```

Comment:
I thought we put all access related stuff to ./access

Reply:
Moved to the "access" folder